### ISSUE TITLE
[Launcher/Installer] Suggestion - Bundle symbol font with the applications (for better compatibility with MacOS/Linux)

### ISSUE THREAD

**Issue by refringe** [sp-tarkov/server#223]
## **General information**
**Project issue is for:**
Launcher/Installer

## Description

Currently the installer & launcher applications depend on the font "Segoe MDL2 Assets" to be installed locally on the users systems. 
Since this font is only available on Windows & redistribution is not allowed, this will break icons like the close & minimize buttons for MacOS / Linux users. 

[There are ways](https://dev.sp-tarkov.com/MadByte/Linux-Guide/src/branch/main/docs/issues.md#13-missing-fonts--icons) to install said font on those operating systems, but usage is not permitted by Microsoft and also requires the user to manually find and install the font if they want to fix it. 

Currently, only three symbols are affected by this: Close, Minimize, Settings menu (Go Back).

My suggestion would be to either use static *.svg / *.png icons or switch to bundling a more openly licensed symbol font like the [Material Icons font](https://github.com/google/material-design-icons/tree/master/variablefont) which is licensed under the [Apache 2.0 License](https://github.com/google/material-design-icons/blob/master/LICENSE).

In the end, this is not a hard requirement for playing the mod on MacOS/Linux - but it would be a nice to have non the less.

## Screenshots

---
### Attachments
- [issue_icons1.jpg](https://dev.sp-tarkov.com/attachments/3e526a45-9252-4ddb-ac6a-474fd3a51555) (1 KiB)
- [issue_icons2.jpg](https://dev.sp-tarkov.com/attachments/65daf6ac-2b57-4fab-bd74-da0246fb8b35) (1 KiB)

---

[Originally written](https://dev.sp-tarkov.com/SPT/Issues/issues/732) by _**MadByte**_

**Comment by refringe** [comment]
Disclaimer: I've no experience in application development, nor any in-depth knowledge of AvaloniaUI or C# coding in general.

I've tried a couple ways to fix this on my end for fun - it works pretty well and looks almost the same when using a trimmed down version of the deprecated `Material Icons Regular` font (348kb to 1,8kb) as fallback to the MDL2 font, so size shouldn't be a concern.

But that way it would still require you to add a proper license & also mean that a third-party asset would need to be added to the repo to fix something occurring on OSes you're not actually supporting.

I tried to use [AvaloniaUI Fluent Icons](https://avaloniaui.github.io/icons.html) using the `PathIcon` functionality to work around the licensing issue - just like the `Gear` icon in the title bar - but I couldn't get them to use the same styling/color as the fonts by default and when hovered, they just stayed white.

Another thought was to use CCO / Unlicense licensed icons & convert them into a very own icon font for convenience of use, but at this point I started to realize that this is getting ridiculous for an X, a horizontal line & an arrow pointing right... :joy:

As I've said, this is really a non-issue apart from it being ugly to look at, so if there's a chance to get this solved in the future, I'd appreciate it and would offer to open a PR at some point. I'd just need a short statement on your thoughts.

Thanks for your time, cheers!


---
[Originally written](https://dev.sp-tarkov.com/SPT/Issues/issues/732#issuecomment-27222) by _**MadByte**_
